packages/theming/atlas/scripts/build.js

```diff
@@ -4,12 +4,26 @@ const concurrently = require("concurrently");
 
 const mode = process.argv[2] || "production";
 const MX_PROJECT_PATH = process.env.ATLAS_MX_PROJECT_PATH; // should be an absolute path.
-const outputDir =
-    mode !== "production" && MX_PROJECT_PATH ? join(MX_PROJECT_PATH, "theme") : join(__dirname, "../dist/theme");
-const projectDeployDir = mode !== "production" && MX_PROJECT_PATH ? join(MX_PROJECT_PATH, "deployment/web") : null;
+let outputDir;
+let projectDeployDir;
+
+switch (mode) {
+    case "test":
+        outputDir = join(__dirname, "../tests/testProject/theme");
+        break;
+    case "production":
+        outputDir = join(__dirname, "../dist/theme");
+        break;
+    case "development":
+        outputDir = MX_PROJECT_PATH ? join(MX_PROJECT_PATH, "theme") : join(__dirname, "../dist/theme");
+        projectDeployDir = MX_PROJECT_PATH ? join(MX_PROJECT_PATH, "deployment/web") : null;
+        break;
+    default:
+        throw new Error("No mode specified. Exiting now...");
```

Comment:
IIUC, this default will never be executed, based on the expression on line 5.

Reply:
I wanted to make the code future proof, we could remove indeed if you feel strong on this one

Reply:
It's common practice for CLI's to pick default values for options, so I think we will always assign a mode in the future. This code will likely stay untouched in the future, so I would remove it.